spec/PhpSpec/Formatter/TapFormatterSpec.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+namespace spec\PhpSpec\Formatter;
+
+use PhpSpec\Console\IO;
+use PhpSpec\Event\SuiteEvent;
+use PhpSpec\Event\ExampleEvent;
+use PhpSpec\Event\SpecificationEvent;
+use PhpSpec\Loader\Node\ExampleNode;
+use PhpSpec\Loader\Node\SpecificationNode;
+use PhpSpec\Formatter\Presenter\PresenterInterface;
+use PhpSpec\Listener\StatisticsCollector;
+use PhpSpec\ObjectBehavior;
+use Prophecy\Argument;
+
+class TapFormatterSpec extends ObjectBehavior
+{
+    function let(PresenterInterface $presenter, IO $io, StatisticsCollector $stats)
+    {
+        $this->beConstructedWith($presenter, $io, $stats);
+    }
+
+    function it_is_an_event_subscriber()
+    {
+        $this->shouldHaveType('Symfony\Component\EventDispatcher\EventSubscriberInterface');
+    }
+
+    function it_outputs_version_on_beforesuite_event(SuiteEvent $event, IO $io)
+    {
+        $this->beforeSuite($event);
+        $expected = 'TAP version 13';
+        $io->writeln($expected)->shouldHaveBeenCalled();
+    }
+
+    function it_outputs_plan_on_aftersuite_event(SuiteEvent $s_event, ExampleEvent $e_event, ExampleNode $example, IO $io, StatisticsCollector $stats)
+    {
+        $stats->getEventsCount()->willReturn(3);
+        $e_event->getExample()->willReturn($example);
+        $example->getTitle()->willReturn('foobar');
```

Comment:
It's there to prime the pump, i.e. the 1 in 1..3. Would print zero otherwise which should be invalid TAP ergo technically correct and technically me being pedantic ;).